app/src/main/java/com/umc/anddeul/home/PostWriteActivity.kt

```diff
@@ -97,15 +100,14 @@ class PostWriteActivity : AppCompatActivity() {
         }
 
         // 이미지 파일들을 MultipartBody.Part로 변환
-        val imageParts: List<MultipartBody.Part> = copiedImagesUri.map { uri ->
+        val imageParts: List<MultipartBody.Part> = selectedImagesUri.map { uri ->
```

Comment:
이거 그냥 지역 변수로 분리 안하고 한줄로 합쳐도 되겠는걸요 ㅎㅎ
```suggestion
        val imageParts: List<MultipartBody.Part> = selectedVPAdapter.getImages().map { uri ->
```